Installing-and-Configuring/OS/index.md

```diff
@@ -58,19 +58,27 @@ Your other partitions may be encrypted or not based on your preferences and this
 
 ### Using TPM for decryption of drives
 
-Heads can protect a LUKS Disk Unlock Key slot using the TPM. This will ensure that upon booting with Heads, only a TPM Disk Unlock Key passphrase with the proper TPM measurements (PCRs states) can access the data on the root partition. The state currently measured are the LUKS header itself (slots existing), firmware measurements (refer to https://osresearch.net/Keys/#tpm-pcrs).
+Heads can protect LUKS encrypted content using the TPM to release a decryption key when provided with proper measurements and passphrase. This ensures that upon booting with Heads, only a TPM Disk Unlock Key passphrase with the proper TPM measurements (PCRs measured states of firmware content) can release the Disk Unlock decryption key, fed through an additional initrd file being injected into OS boot process for it to unlock encrypted filesystems prior of continuing its boot process. The states verified prior or releasing the Disk Unlock key into slot 1 are measurements of the LUKS header itself (dump from cryptsetup header output), firmware measurements (refer to https://osresearch.net/Keys/#tpm-pcrs), in addition to the Disk Unlock Key passphrase.
```

Comment:
The states verified prior of releasing the Disk Unlock key into slot 1 are measurements of the LUKS header itself (dump from cryptsetup header output measured in PCR6), firmware measurements (refer to https://osresearch.net/Keys/#tpm-pcrs), if provided by the good Disk Unlock Key passphrase.

Reply:
updated.  will push in a minute.